packages/e2e-tests/specs/experiments/block-directory-add.test.js

```diff
@@ -47,6 +47,24 @@ const MOCK_BLOCK1 = {
 	humanized_updated: '5 months ago',
 };
 
+const MOCK_PLUGIN = {
```

Comment:
We could maybe update the name of this variable to communicate that it's the WordPress plugin object post install otherwise it's a bit confusing as to how this differs from the block object until you crawl through the code to see where its returned.

Maybe `MOCK_INSTALLED_BLOCK_DETAILS`.. ?

Reply:
I can see how that is confusing. What about `MOCK_INSTALLED_BLOCK_PLUGIN_DETAILS` instead since we are actually returning the plugin object, not the block details?

Reply:
👍 Good call.

Reply:
Updated in b6689eb.